test/func_memory_uma.py

```diff
@@ -0,0 +1,168 @@
+BATCHFILE = """\
+c:\\%s
+rem end
+"""
+
+CONFIG = """\
+$_hdimage = "dXXXXs/c:hdtype1 +1"
+$_floppy_a = ""
+"""
+
+
+def memory_uma_strategy(self):
+
+# Note: Not sure if I need this
+    if 'FDPP' in self.version:
+        self.mkfile("userhook.sys", """DOS=HIGH\n""", newline="\r\n")
```

Comment:
What does this do?

Reply:
I think its not needed, not testing HMA
here.

Reply:
Okay, cool I'll remove it.

BTW, this test passes on DR-DOS 7.01, MS-DOS 6.22, MS-DOS 7.0, MS-DOS 7.1, and FDPP, but it **fails** on FreeDOS 1.2. Is that expected?

Reply:
It seems to work if not use `-dumb`.
UMB/HMA support in freedos is very,
very rudimentary as we know from
all the rewrites that were done.
So while not expected, its also not
very surprising.

Reply:
Shouldn't we move to freedos 1.3?
I think it was quite active (esp in
back-porting fdpp patches), so maybe
more tests will work?

Reply:
> Shouldn't we move to freedos 1.3?

Yes it's on my personal list of things to do.